### DISCUSSION TITLE
How i can to close an application on system tray with shortcuts?

### DISCUSSION THREAD

**Discussion by 2024-jcmljunior** [i3/i3#4924] [Q&A]
Hello, how could I close a minimized application?
I have the system tray enabled and it would be very useful to have a shortcut to do this job.

This way I can close most things with the exception of applications that use the system tray.
`bindsym $MODKEY1+Q kill`

I need an function/shortcut for close applications on system tray.

**Comment by Airblader** [comment]
The only generic way I can think of is `pkill`. There is no generic way of closing an application from its tray icon.

**Reply by 2024-jcmljunior** [reply]
I understand but this command requires parameters to work.
Is it possible to know the identifier of the window in focus to end it with this command?
Maybe it's a case of a new feature.

**Reply by Airblader** [reply]
Tray icons are never focused, none of i3bar is ever focused.

**Reply by 2024-jcmljunior** [reply]
I understand. thank you anyway.
